src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1515,14 +1515,15 @@ int xclLogMsg(xclDeviceHandle handle, xrtLogMsgLevel level, const char* tag, con
 {
     va_list args;
     va_start(args, format);
-    xocl::shim *drv = xocl::shim::handleCheck(handle);
-    int ret = drv ? drv->xclLogMsg(level, tag, format, args) : -ENODEV;
+    //Do NOT add device handle check
```

Comment:
maybe we want to introduce a new api and remove handle from arg list and mark this one obsolete?

Reply:
How do we ensure that it will make it to common code for aws also? Right now this is missing from aws code. This API is now used in emulation code also. So changing it and removing device handle impacts them also.

Reply:
after we remove aws version of shim.cpp, we'll use our version of shim.cpp where the api has already been available. i mean add a new one without device handle. i did not mean changing existing one.

Reply:
ok.  I will make that change.